tests/embed/test_mpca.py

```diff
@@ -47,7 +47,7 @@ def test_mpca(var_ratio, n_components):
     testing.assert_equal(x0_proj.ndim, 2)
     testing.assert_equal(x0_proj.shape[0], 1)
     testing.assert_equal(x0_proj.shape[1], n_components)
-    x0_rec = mpca.inverse_transform(x0_proj)
+    x0_rec = mpca.inverse_transform(x0_proj.reshape(-1))
```

Comment:
A bit difficult to review. Can you explain a bit why the change?

Reply:
Before: the shape of `x0_proj` is (1, n_features). After: the shape of `x0_proj` is (n_features,). Here we test whether a vector can be reconstructed as tensor in original shape. It is also correct. I made the change here simply to cover [line 249](https://github.com/pykale/pykale/blob/master/kale/embed/mpca.py#L249) of mpca.py.